internal/mapper/attrmapper/list_nested.go

```diff
@@ -50,6 +51,16 @@ func (a *ResourceListNestedAttribute) ApplyNestedOverride(path []string, overrid
 	return a, err
 }
 
+func (a *ResourceListNestedAttribute) NestedMerge(path []string, attribute ResourceAttribute, intermediateComputability schema.ComputedOptionalRequired) (ResourceAttribute, error) {
+	var err error
+	a.NestedObject.Attributes, err = a.NestedObject.Attributes.MergeAttribute(path, attribute, intermediateComputability)
+	if err == nil {
+		a.ComputedOptionalRequired = intermediateComputability
+	}
```

Comment:
I'm assuming the intention here is to bubble up the computability? So if a required parameter is aliased into a nested list that is computed, the nested list is marked as required?

Would like to have a unit test that codifies this behavior